x-pack/legacy/plugins/index_management/__jest__/client_integration/helpers/template_form.helpers.ts

```diff
@@ -39,6 +49,28 @@ export const formSetup = async (
     testBed.find('submitButton').simulate('click');
   };
 
+  const clickEditFieldAtButton = (index: number) => {
```

Comment:
Nit: `clickEditButtonAtField` might read a little more clearly. Otherwise it sort of sounds like we're clicking a field located at a button. Same idea regarding `clickRemoveFieldAtButton`.

Reply:
good point! fixed.